src/book.js

```diff
@@ -570,7 +574,7 @@ export class Book {
     block.forEach((page, i) => {
       if (page == 'b' || page === undefined) {
         // blank page, move on.
-      } else {
+      } else if (page instanceof PDFEmbeddedPage) {
```

Comment:
interesting -- good idea

minor nit/ask (but feel free to disregard) -- could we `console.error` in an `else` off of this `if/else if` to keep an eye out in case anything else weird slips in there?

Reply:
sure! and lol I put that in there mostly to keep the static checker from complaining